doc/docportal/advanced_topics/autostart.rst

```diff
@@ -0,0 +1,22 @@
+=========================
+Autostart
+=========================
+
+Use the autostart feature to automatically start the game located in the same folder as the ScummVM executable, as long as it is a game supported by ScummVM. It is the equivalent of using the ``--path`` and ``--auto-run`` :doc:`command line options <../advanced_topics/command_line>`. 
+
+There are two ways to use the autostart feature:
+
+- Rename the executable file to ``scummvm-auto.xxx``, where ``xxx`` is the file type extension. 
```

Comment:
Maybe it would be worth mentioning that on macOS the executable to rename is `ScummVM.app/Contents/MacOS/scummvm` and not the `ScummVM.app` application? As written here that seems ambiguous for the macOS case.